docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -748,6 +750,41 @@ In this example, the table is partitioned by the month of ``order_date``, a hash
         country VARCHAR)
     WITH (partitioning = ARRAY['month(order_date)', 'bucket(account_number, 10)', 'country'])
 
+Sorted tables
+------------------
+
+In addition to partitioning, Iceberg tables may have a specified list of sort columns. When writes
+are made to the table, each file will individually be sorted by these columns. Sorting can be
+particularly helpful when selective reads are frequently made against a column with a high cardinality.
+
+Sort orders are provided using the ``sorted_by`` table property. For example::
+
+    CREATE TABLE iceberg.testdb.customer_orders (
+        order_id BIGINT,
+        order_date DATE,
+        account_number BIGINT,
+        customer VARCHAR,
+        country VARCHAR)
+    WITH (partitioning = ARRAY['country'], sorted_by = ARRAY['order_date'])
```

Comment:
If partitioning isn't strictly required for sorting, let's have the first example not use it.

Reply:
agreed